### PR TITLE
Adding a count down timer to the Home Screen

### PR DESCRIPTION
**Pull request by Darcy382**

### CODE REVIEW

File: lib/card_items/countdown_timer_card.dart [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import 'dart:async';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+final hackathonDate = DateTime.utc(2022, 4, 8, 14);
```
Comment (Darcy382) [inline lib/card_items/countdown_timer_card.dart:5]: @rsquared226 I forgot what time you said I should set this too

Comment (patel-kalp) [reply lib/card_items/countdown_timer_card.dart:5]: April 8th 2pm EST 👺😘🥸🥳🤓🥶😴🥱😵💩👽🤲💋🗣👁👄👁👣👩‍🦳🕵️‍♂️👨🏼‍💻🦹‍♀️🧖‍♀️💆🏾‍♂️👩‍❤️‍💋‍👨👨‍👨‍👧‍👧👩‍👧‍👧👩‍👦‍👦👩‍👦‍👦👙🌂

**Review by rsquared226 — CHANGES_REQUESTED** [review]
I think we have to add 5 hours to the current countdown time, just checked on [this website](https://www.timeanddate.com/date/durationresult.html?m1=2&d1=9&y1=2022&m2=4&d2=8&y2=2022&h1=16&i1=45&s1=4&h2=14&i2=0&s2=0).

Other than that, it works beautifully!

**Comment by Darcy382** [comment]
> I think we have to add 5 hours to the current countdown time, just checked on [this website](https://www.timeanddate.com/date/durationresult.html?m1=2&d1=9&y1=2022&m2=4&d2=8&y2=2022&h1=16&i1=45&s1=4&h2=14&i2=0&s2=0).
> 
> Other than that, it works beautifully!

Ah that must be from the UTC-5 offset.  Just updated it

**Review by rsquared226 — APPROVED** [review]
Niceee once someone else with iOS approves we can merge

**Review by QuinnB11 — APPROVED** [review]
There are two loading wheels when I log in but everything else seems to be working perfectly!

**Pull request merged**